account_reconcile_oca/models/account_reconcile_abstract.py

```diff
@@ -95,7 +95,7 @@ def _get_reconcile_line(
             if line.partner_id
             else False,
             "date": fields.Date.to_string(line.date),
-            "name": line.name,
+            "name": line.name or line.move_id.name,
```

Comment:
But `move_id.name` is the journal entry number, not the ref, so I don't think it's the correct solution.

Reply:
But what we need is the label of the statement line, not that name.

Reply:
Please check how it's done on 15- and on enterprise to compare. I'm not sure this should have the same label as the counterpart.

Reply:
The above video is the behavior of **v15**. The behavior of enterprise in v16 is as follows (it does not define anything in the account.move.line).

![ejemplo](https://github.com/user-attachments/assets/8d63605c-0353-4f93-b6eb-13b01d1e3f37)

Reply:
v18 enterprise example.

![ejemplo](https://github.com/user-attachments/assets/a5881095-2c54-426e-a79a-127d2c8904cc)